ompi/mpi/c/wtime.c

```diff
@@ -40,10 +41,17 @@ double MPI_Wtime(void)
 {
     double wtime;
 
+    /*
+     * See https://github.com/open-mpi/ompi/issues/3003
+     * For now we are forcing the use of gettimeofday() until we find a
+     * more portable solution.
+     */
+#if 0
 #if OPAL_TIMER_CYCLE_NATIVE
     wtime = ((double) opal_timer_base_get_cycles()) / opal_timer_base_get_freq();
 #elif OPAL_TIMER_USEC_NATIVE
     wtime = ((double) opal_timer_base_get_usec()) / 1000000.0;
+#endif
 #else
     /* Fall back to gettimeofday() if we have nothing else */
```

Comment:
@hjelmn + me suggest putting something like:

```c
#if _LINUX_ && OPAL_HAVE_CLOCK_GETTIME
// ... use clock_gettime(..) ...
#else
// ... use gettimeofday(..) ...
#endif
```

That way, we don't need a new configure test -- we *only* use `clock_gettime` if a) we're on Linux, and b) our existing configure test says that we have `clock_gettime` (i.e., we sidestep the whole OS X problem).